.github/workflows/post-deploy.yml

```diff
@@ -22,15 +22,19 @@ on:
         required: false
         default: false
         type: boolean
+      app_name:
+        required: true
+        default: false
```

Comment:
No default for app_name, because it's required - meaning you _have_ to pass a value through